### PR TITLE
Add NftIdToBech32

### LINKED ISSUE #1430: [Task]: Add nftIdToBech32 to message interface and bindings
### Task description

Similar to aliasIdToBech32 here https://github.com/iotaledger/iota.rs/pull/1428/files just for an NFT id

### Requirements

Added to:
- [x] message interface
- [x] java
- [x] python
- [x] nodejs

### Acceptance criteria

-

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add NftIdToBech32

## Links to any relevant issues

Fixes https://github.com/iotaledger/iota.rs/issues/1430

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by samuel-rufi — DISMISSED** [review]

File: client/bindings/java/lib/src/main/java/org/iota/Client.java [hunk 1]
```diff
@@ -542,6 +542,18 @@ public String aliasIdToBech32(AliasId aliasId, String bech32Hrp) throws ClientEx
...
 
+    /**
+     * Converts an nft id to a bech32 address.
+     *
+     * @param nftId The nft id to be converted.
+     * @param bech32Hrp The bech 32 human readable part to use.
+     * @return The bech32 address.
+     * @throws ClientException on error.
+     */
+    public String nftIdToBech32(NftId nftId, String bech32Hrp) throws ClientException {
```
Comment (thibault-martinez) [inline client/bindings/java/lib/src/main/java/org/iota/Client.java:553]: Java changelog

File: client/bindings/nodejs/lib/Client.ts [hunk 2]
```diff
@@ -688,6 +688,21 @@ export class Client {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Transforms an nft id to a bech32 encoded address.
+     */
+    async nftIdToBech32(nftId: string, bech32Hrp?: string): Promise<string> {
```
Comment (thibault-martinez) [inline client/bindings/nodejs/lib/Client.ts:694]: .change

File: client/CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - `alias_id_to_bech32()` to utils;
+- `nft_id_to_bech32()` to utils;
+- `Response::Bech32Address()` to message interface;
```
Comment (thibault-martinez) [inline client/CHANGELOG.md:28]: Maybe also add the message variants ?

File: client/bindings/java/CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-rc.2 - 202x-xx-xx
+
+### Added
+
+- `Client::nftIdToBech32()`;
```
Comment (thibault-martinez) [inline client/bindings/java/CHANGELOG.md]: Was the alias one already released or should it be added here as well ?

Comment (Thoralf-M) [reply client/bindings/java/CHANGELOG.md]: Wasn't, added

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**